.github/prompts/tspconfig_add_missing_languages.prompt.md

```diff
@@ -0,0 +1,14 @@
+---
+mode: agent
+model: Claude Sonnet 4
```

Comment:
ohh I didn't know we could do this!!!!!!!

Reply:
Yeah, it is probably the best feature and it actually works! :)